gamepad/src/gamepad.java

```diff
@@ -0,0 +1,63 @@
+import com.studiohartman.jamepad.ControllerManager;
+import com.studiohartman.jamepad.ControllerState;
+import org.eclipse.paho.client.mqttv3.*;
+
+import java.util.UUID;
+
+public class gamepad {
+
+
+
+
+    public static void main(String[] args) throws MqttException {
+
+        String publisherId = UUID.randomUUID().toString();
+        IMqttClient publisher = new MqttClient("tcp://localhost:1883",publisherId);
```

Comment:
Is it really needed to create a new UUID as a publisherID every time we run the program? 

Isn't it better to just stick to the same one since we're not using a public broker?

Reply:
I'd say we can stick with it for now. Even though we probably won't use a public broker i don't see the harm in using a UUID.